src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -36,7 +35,7 @@ setTimeout(async function () {
       const payload = JSON.parse(message.data);
       const action = {
         'UpdateStaticFile': () => updateStaticFile(payload.path),
-        'BlazorHotReloadDeltav1': () => applyBlazorDeltas(payload.sharedSecret, payload.deltas),
```

Comment:
@pranavkm please see if this file's merge is correct. There are 2 other places, I used what's in release/6.0.1xx. If this does not look right, you could remake this merge. The only complex merge is this file.